modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -612,13 +625,24 @@ public void testAddObjectAction() throws Exception {
 				originalPermissionChecker);
 		}
 
+		ObjectAction systemObjectAction = _addObjectAction(
+			name, ObjectActionExecutorConstants.KEY_WEBHOOK,
+			ObjectActionTriggerConstants.KEY_ON_AFTER_ADD,
+			UnicodePropertiesBuilder.put(
+				"secret", "onafteradd"
+			).put(
+				"url", "https://onafteradd.com"
+			).build(),
+			false);
```

Comment:
I think we could add an Object Action and see that if the action is working.
Like the previous examples.

Reply:
we can talk if you want.